### ISSUE TITLE
INES: Jaws

### ISSUE THREAD

**Issue by Kevin-Woods-Tessella** [ISISComputingGroup/IBEX#3523]
As a developer, I need to confirm that the standard IBEX jaws IOC & OPI can control the INES jaws.

The jaws on INES are manufactured by an Italian company.  The jaws motors are currently controlled by Galil.  Under SECI, the jaws on INES were controlled by a VI called the "GEM Jaws" (which refers to the jaws on GEM prior to their recent upgrade).  We should check that the jaws on INES can be controlled by the standard IBEX jaws IOC & OPI.  The expectation is that the standard IBEX jaws IOC & OPI will be suitable; the purpose of this ticket is to avoid a late surprise.

#### Acceptance Criteria
1. Confirm that the standard IBEX jaws IOC & OPI can control the INES jaws.
   1. If the standard IBEX jaws IOC & OPI are suitable, no further action is required.
   1. If the standard IBEX jaws IOC & OPI are not suitable, create a follow-on ticket to implement a suitable jaws IOC & OPI.

#### Notes
1. [Picture of INES jaws (and fixed monitor)](http://www.facilities.rl.ac.uk/isis/computing/ICPdiscussions/INES/INES_Jaws_and_Fixed_Monitor.jpg).

**Comment by ThomasLohnert** [comment]
Standard IBEX jaws should be suitable.
